tests/benchmarks/wait_queues/testcase.yaml

```diff
@@ -0,0 +1,21 @@
+common:
+  tags:
+    - kernel
+    - benchmark
+  integration_platforms:
+    - qemu_x86
+    - qemu_arc/qemu_arc_em
```

Comment:
Why only those platforms?  In particular seeing ARM missing feels weird.

Reply:
No particular reason aside form a copy-paste from another benchmark project.  Updated to use an ARM and x86.